llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1666,23 +1666,22 @@ void SpirvLowerGlobal::lowerBufferBlock() {
           if (!isa<Instruction>(user))
             continue;
 
-          GetElementPtrInst *getElemPtr = dyn_cast<GetElementPtrInst>(user);
+          // Skip all bitcasts, looking for a GEP.
+          Value *inst = user;
```

Comment:
Could we use `stripPointerCasts`? If all-zero GEPs would be the issue, maybe we can create a free function, something like `Value* stripBitcasts(Value *val, SmallVectorImpl<BitCastInst>& stripped)`